src/main/java/com/algorand/algosdk/v2/client/model/TransactionsResponse.java

```diff
@@ -0,0 +1,101 @@
+package com.algorand.algosdk.v2.client.model;
+
+import java.util.List;
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class TransactionsResponse {
+
+	/**
+	 * Round at which the results were computed. 
+	 */
+	private long currentRound;
+	private boolean currentRoundIsSet;
+	@JsonProperty("current-round")
+	public void setCurrentRound(long currentRound){
```

Comment:
We don't need setters for these. The only one we may want a setter for is `TransactionParams`

Reply:
The setters are all removed.